lib/modules/dosomething/dosomething_reportback/dosomething_reportback.query.inc

```diff
@@ -66,11 +66,19 @@ function dosomething_reportback_build_reportbacks_query($params = array()) {
   }
 
   if (isset($params['runs'])) {
-    if (is_array($params['runs'])) {
```

Comment:
not sure why we're removing this. lets chat in Slack about this when you get a chance - i’m a little confused exactly on purpose of this PR and why we need to check the id’s passed are numeric..  refs https://github.com/DoSomething/phoenix/pull/6359#issuecomment-209046454